json_formatter.go

```diff
@@ -43,6 +54,21 @@ type JSONFormatter struct {
 	// }
 	FieldMap FieldMap
 
+	// LogLevelMap allows users to customize the values used for the various log levels
+	// As an example:
+	// formatter := &JSONFormatter{
+	//   	LogLevelMap: LogLevelMap{
+	// 		 PanicLevel:  1,
```

Comment:
Please make the example valid. Better yet, make an `ExampleLevelMap` function in the [test](https://golang.org/pkg/testing/#hdr-Examples) file.